src/test/java/seedu/foodrem/storage/JsonSerializableFoodRemTest.java

```diff
@@ -52,14 +52,6 @@ public void toModelType_duplicateItems_throwsIllegalValueException() throws Exce
                 dataFromFile::toModelType);
     }
 
-    @Test
-    public void toModelType_duplicateTags_throwsIllegalValueException() throws Exception {
-        JsonSerializableFoodRem dataFromFile = JsonUtil.readJsonFile(DUPLICATE_TAG_FILE,
-                JsonSerializableFoodRem.class).get();
-        assertThrows(IllegalArgumentException.class, JsonSerializableFoodRem.MESSAGE_DUPLICATE_TAGS,
-                dataFromFile::toModelType);
-    }
-
```

Comment:
We shall allow duplicate tags in the data file. During conversion these tags will be removed